### PR TITLE
Added SpliceAI Version 1.3

### PR DESCRIPTION
**Pull request by kishorejaganathan**
`conda build --numpy 1.14 -c bioconda spliceai` will produce the appropriate tar.bz2 file.

Summary of changes:
- moved the existing meta.yaml file (for version 1.2.1) into a new folder named 1.2.1
- created a new meta.yaml file for version 1.3
- added license files

Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
I merged the bot's version in before I saw yours. It turns out that more recent keras versions require a more recent tensorflow, so I've updated the recipe in the bot's PR accordingly.

**Comment by kishorejaganathan** [comment]
There is a python >= 3 restriction added by the person who created this yaml file, which is not required (SpliceAI runs on both python 2 and python 3). Could you remove that restriction so that users with python 2 environments will be able to install SpliceAI as well? Thank you very much.

**Comment by dpryan79** [comment]
sure
